elastic/apm-server
# Add app_logs to apmpackage.

## Motivation/summary
Backports parts of https://github.com/elastic/apm-server/pull/6791 to introduce an apm specific "logs-*" datastream.

## How To Test
Run a local EPR with this package and point your Kibana instance to it. 
Once this is merged and released to staging, we can follow along with Reproduction steps from #7074 and ensure this also fixes the issue on cloud. 

## Related issues
fixes #7074
